src/main/scala/io/iohk/ethereum/utils/StringUtils.scala

```diff
@@ -1,9 +1,8 @@
-package io.iohk.ethereum.mallet.common
+package io.iohk.ethereum.utils
 
 import akka.util.ByteString
-import org.bouncycastle.util.encoders.Hex
 
-object StringUtil {
+object StringUtils {
 
   def unquote(s: String): String = {
```

Comment:
Can we get rid of this method, since we already touching this code?
It is unused and it throws exceptions without saying so in the type signature.

Reply:
sure, thank you for this detail!

Reply:
I see you deleted other unused stuff - nice catch! 👍